### PR TITLE
Improve CGMES documentation regarding transformers and boundaries

### PR DESCRIPTION
**Pull request by miovd**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**
Doc update: complete documentation regarding CGMES import of transformers and equipment at boundary

### CODE REVIEW

**Review by So-Fras — CHANGES_REQUESTED** [review]
Thank you again for the documentation work. Most of my remarks are about typos.

File: pages/documentation/grid/formats/cim-cgmes.md [hunk 1]
```diff
@@ -388,7 +421,15 @@ If the CGMES `Switch` has its ends both inside the same voltage level, it is map
...
+- If there is exactly one other linking equipment linked to this `TopologicalNode` in another `SubGeographicalRegion`, they are both mapped to PowSybl [`HalfLines`](../model/index.md#half-line), part of the same PowSyBl [`TieLine`](../model/index.md#tie-line).
+- If there are two or more other linking equipments linked to this `TopologicalNode` in different `SubGeographicalRegions`:
+  - If there are only two linking equipment with their boundary terminal connected **and** in different `SubGeographicalRegion`, they are both mapped to PowSybl [`HalfLines`](../model/index.md#half-line), part of the same PowSyBl [`TieLine`](../model/index.md#tie-line) and all other `EquivalentBranches` are mapped to PowSyBl [`DanglingLines`](../model/index.md#dangling-line).
+  - Otherwise, they are all mapped to PowSyBl [`DanglingLines`](../model/index.md#dangling-line).
+
+If the CGMES `Switch`  is mapped to a PowSybl [`DanglingLine`](../model/index.md#dangling-line), its attributes are as described below:
```
Comment (colinepiloquet) [inline pages/documentation/grid/formats/cim-cgmes.md:432]:
```suggestion
If the CGMES `Switch` is mapped to a PowSyBl [`DanglingLine`](../model/index.md#dangling-line), its attributes are as described below:
```

File: pages/documentation/grid/formats/cim-cgmes.md [hunk 2]
```diff
@@ -388,10 +421,18 @@ If the CGMES `Switch` has its ends both inside the same voltage level, it is map
...
+- If there is exactly one other linking device linked to this `TopologicalNode` in another `SubGeographicalRegion`, they are both mapped to PowSybl [`HalfLines`](../model/index.md#half-line), part of the same PowSyBl [`TieLine`](../model/index.md#tie-line).
+- If there are two or more other linking devices linked to this `TopologicalNode` in different `SubGeographicalRegions`:
+  - If there are only two linking devices with their boundary terminal connected **and** in different `SubGeographicalRegion`, they are both mapped to PowSyBl [`HalfLines`](../model/index.md#half-line), part of the same PowSyBl [`TieLine`](../model/index.md#tie-line) and all other `EquivalentBranches` are mapped to PowSyBl [`DanglingLines`](../model/index.md#dangling-line).
+  - Otherwise, they are all mapped to PowSyBl [`DanglingLines`](../model/index.md#dangling-line).
+
+If the CGMES `Switch` is mapped to a PowSyBl [`DanglingLine`](../model/index.md#dangling-line), its attributes are as described below:
+- `R`, `X`, `G`, `B` are `0.0`;
+- `UcteXnodeCode` is copied from the name of the `TopologicalNode` or the `ConnectivityNode` (respectively in `NODE-BREAKER` or `BUS-BRANCH`) inside boundaries;
```
Comment (annetill) [inline pages/documentation/grid/formats/cim-cgmes.md:434]: Pairing key here.

**Comment by So-Fras** [comment]
@colinepiloquet I know you have transfered most of the cim-cgmes documentation in readthedocs. What about this PR? Could you check if there is something to get from it and then close it?
Thanks a lot!

**Comment by annetill** [comment]
> @colinepiloquet I know you have transfered most of the cim-cgmes documentation in readthedocs. What about this PR? Could you check if there is something to get from it and then close it? Thanks a lot!

Indeed I don't like "linking device".

**Comment by colinepiloquet** [comment]
> > @colinepiloquet I know you have transfered most of the cim-cgmes documentation in readthedocs. What about this PR? Could you check if there is something to get from it and then close it? Thanks a lot!
> 
> Indeed I don't like "linking device".

Me neither. We do need to integrate this PR in the documentation though. I have trouble finding another word for linking device.

**Comment by colinepiloquet** [comment]
I have done some corrections, according to your feedback @So-Fras and @annetill. Once merged, I will transfer it to the documentation on readthedocs.

File: pages/documentation/grid/formats/cim-cgmes.md [hunk 3]
```diff
@@ -207,7 +207,7 @@ If the `ACLineSegment` is mapped to a PowSyBl [`DanglingLine`](../model/index.md
 - `X` is copied from CGMES `x`
 - `G` is copied from CMGES `gch` if defined, `0.0` otherwise
 - `B` is copied from CGMES `bch`
-- `UcteXnodeCode` is copied from the name of the `TopologicalNode` or the `ConnectivityNode` (respectively in `NODE-BREAKER` or `BUS-BRANCH`) inside boundaries
+- `ParingKey` is copied from the name of the `TopologicalNode` or the `ConnectivityNode` (respectively in `NODE-BREAKER` or `BUS-BRANCH`) inside boundaries
```
Comment (So-Fras) [inline pages/documentation/grid/formats/cim-cgmes.md:210]: Small typo: "Pairing"

**Review by So-Fras — CHANGES_REQUESTED** [review]

**Review by So-Fras — APPROVED** [review]

**Pull request merged**